README.md

```diff
@@ -143,3 +143,12 @@ Mapping for Paradise station can be daunting to new contributors. Here's a compr
 
 >All other non-code assets, including icons and sound files, are licensed under the [Creative Commons 3.0 BY-SA license](https://creativecommons.org/licenses/by-sa/3.0/), unless otherwise specified within the folder or file.
 </details>
+
+<details>
+
+<summary><a href="#"><img src="https://img.shields.io/badge/licence-CC_4.0_BY--NC-blue?style=for-the-badge" alt="Creative Commons 4.0 BY-NC" align="centre"></a></summary>
+
+>Any files with the ancestor directories [`Paradise/icons/tgmc`](icons/tgmc) are licensed under the [Creative Commons 4.0 BY-NC license](https://creativecommons.org/licenses/by-nc/4.0).
+>
+>Further files or folders may also fall under this license, and any such instances will be specified within the folder or file.
+</details>
```

Comment:
Put this below the goon `Creative Commons 3.0 BY-NC-SA` license and the `Creative Commons 3.0 BY-SA` license please. The 3.0 BY-SA license says "All other non-code assets" which is awkward if it is not the last license.